docs/test-scenarios/phase1.md

```diff
@@ -0,0 +1,69 @@
+## Submit and vote on constitution governance action
+
+### Test Steps
+
+1. Produce text file and hash digest of the proposed constitution
+2. A regular wallet builds a transaction with an ada deposit, return address including the anchor of proposed constitution
```

Comment:
Under CIP-95 transactions Can also be built by a webapp and passed to the wallet.